x-pack/plugins/triggers_actions_ui/public/application/components/simple_connector_form.tsx

```diff
@@ -162,6 +196,12 @@ const SimpleConnectorFormComponent: React.FC<SimpleConnectorFormProps> = ({
           {index !== secretsFormSchema.length ? <EuiSpacer size="m" /> : null}
         </React.Fragment>
       ))}
+      {configFormSchemaAfterSecrets.map(({ id, ...restConfigSchema }, index) => (
+        <React.Fragment key={`config.${id}`}>
+          <FormRow id={`config.${id}`} {...restConfigSchema} readOnly={readOnly} />
+          {index !== configFormSchemaAfterSecrets.length ? <EuiSpacer size="m" /> : null}
```

Comment:
i think this should be `index !== configFormSchemaAfterSecrets.length - 1`

Reply:
I copy pasta what was above, let me see if we need to change both